interface/main/main_screen.php

```diff
@@ -116,6 +116,10 @@
 if ($GLOBALS['new_tabs_layout']) {
```

Comment:
yep,
Faulty logic on my part. Would need to convert this global to a session and incorporate it throughout codebase. That would be too complicated and not worth making the code murky. Better to only support it on tabs which will document on wiki. Will plan to bring in your code into the codebase as is.